### ISSUE TITLE
Errors when styled-components is not installed

### ISSUE THREAD

**Issue by wescopeland** [South-Paw/react-obfuscate-ts#2]
Trying to use this package in my app, but keep getting this error. We're running on Emotion rather than styled-components.

```
error - ./node_modules/@south-paw/react-obfuscate-ts/dist/react-obfuscate-ts.esm.js:2:0
Module not found: Can't resolve 'styled-components'
```

**Comment by South-Paw** [comment]
Hi! `styled-components` is a peer-dependency and needs to be installed in your project. I'll update the readme so this isn't missed by anyone else. Thank you for raising it :)

**Comment by South-Paw** [comment]
Better late than never - fixed in [2.0.0 release](https://github.com/South-Paw/react-obfuscate-ts/releases/tag/2.0.0), removed styled components as a peer-dependency
